mgmtsystem/hawkular.py

```diff
@@ -210,78 +244,131 @@ def wait_vm_stopped(self, vm_name, num_sec):
     def wait_vm_suspended(self, vm_name, num_sec):
         raise NotImplementedError('wait_vm_suspended not implemented.')
 
-    def list_server_deployment(self, **kwargs):
+    def list_server_deployment(self, feed_id=None):
```

Comment:
Sure, my question was, what if something else is "erroneously" passing parameters that are ignored. With ```**kwargs``` it doesn't matter, but restricting the list will make it break.